nbs/docs/tutorials/21_configure_optimizers.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        NHITS(h=horizon, input_size=input_size, max_steps=100),
</pre><p>can you give these models aliases and then use <code>utilsforecast.plotting.plot_series</code> on the next cell? That will produce a single plot with both models which will make it easier to compare.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1015/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Great!